tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -55,6 +56,45 @@ git add -p config/default.cfg main.gms
 ```
 and then selecting the change in `default.cfg` and the first change to the input data in `main.gms` with `y`, and then ignoring possible other changes in `main.gms` with `d`. Create a pull request to push this change to the main REMIND repository, and REMIND will use the new data by default.
 
+## Adjusting and Enhancing Input Data using madrat
+
+To learn the basics about [madrat](https://pik-piam.r-universe.dev/articles/madrat/madrat.html) and [madrat caching](https://pik-piam.r-universe.dev/madrat/doc/madrat-caching.html), please refer to the corresponding vignettes.
+
+### Remote development on the cluster using VS Code
+
+If you are developing in R using VS Code, you can connect to the cluster via ssh and develop on the cluster remotely. Please check the following tutorial for detailed information:
+
+- [Using R in the cluster with VSCode](https://gitlab.pik-potsdam.de/pascalfu/bettercode-vscode/-/blob/main/R_VScode_in_the_cluster.md)
+- [How to connect VScode to the cluster using ssh](https://github.com/pik-piam/discussions/discussions/4)
```

Comment:
Either explicitly refer people to [using compute nodes instead of login nodes](https://gitlab.pik-potsdam.de/pascalfu/bettercode-vscode/-/blob/main/R_VScode_in_the_cluster.md#a-word-of-caution-on-using-the-cluster) for doing this, or cut this section entirely.
Lest the Eye of ~~Sauron~~ Ciaron turns on the poor little REMIND group for grinding the cluster access to a halt.